source/release-notes/backend.txt

```diff
@@ -13,6 +13,39 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240605:
+
+5 June 2024 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Atlas Events
+
+  - Apps with the Atlas Events feature flag will start to react to changes made in their linked Atlas clusters.
+
+  - On cluster deletion, Apps will delete datasources and all resources linked to that datasource for datasources connected to the deleted cluster.
+
+  - When the cluster is updated (instance size change), the next subsequent usage of a Mongo Client will have small 
+    cold start time caused by refreshing client connection details.
+
+- Improvements to Function Editing Experience: All Apps will now be able to use a new function editor with autocomplete for common functionalities and App Services context. This includes:
+
+  - A new button to allow formatting your functions' code
+
+  - A new clear and run button to replace the Clear result one
+  
+  - Context autocomplete
+
+- Collections in mixed data properties is now availbale. See :ref:`<collections-in-mixed>` for more information. 
+
+- Error logs for GraphQL failures now appear in an app's Logs page.
+
+- Users can no longer disable client max offline time. Users can continue to increase or decrease
+  the number of days for client max offline time where neccessary.
```

Comment:
Is there a ticket to also update this in our docs (e.g. [here](https://www.mongodb.com/docs/atlas/app-services/sync/go-to-production/optimize-sync-atlas-usage/#client-maximum-offline-time) or [here](https://www.mongodb.com/docs/atlas/app-services/sync/go-to-production/optimize-sync-atlas-usage/#client-maximum-offline-time))?

Reply:
Thanks, will make a ticket for this as well! I haven't seen one.